src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+interface Props {
+  goods: string[];
+}
+
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <div>
+      <ul>
+        {goods.map(good => (
+          <li key={good}>
+            {good}
+          </li>
+        ))}
+      </ul>
+    </div>
```

Comment:
do you need this div?
```suggestion
      <ul>
        {goods.map(good => (
          <li key={good}>
            {good}
          </li>
        ))}
      </ul>
```